src/Raven.Server/Documents/DocumentDatabase.cs

```
@@ -571,11 +571,31 @@ public void IncrementalBackupTo(string backupPath)
         }
 
         /// <summary>
-        /// this event is intended for entities that are not singeltons 
+        /// this event is intended for entities that are not singletons 
         /// per database and still need to be informed on changes to the database record.
         /// </summary>
         public event Action DatabaseRecordChanged;
 
+        public void ValueChanged(long index)
+        {
+            try
+            {
+                if (_databaseShutdown.IsCancellationRequested)
+                    ThrowDatabaseShutdown();
+
+                SubscriptionStorage?.HandleDatabaseValueChange();
+            }
+            catch
+            {
+                if (_databaseShutdown.IsCancellationRequested)
```

Why do we have this here? It swallow all exceptions _except_ shutdown?

---

Just blindly copied it from `StateChanged` to be frank. Shouldn't it be like that?

---

Not sure, to be honest. Need to check upward to see what is the behavior. 
Swallowing the exception is probably not okay anyway